src/Utils.ps1

```diff
@@ -118,6 +118,17 @@ function Add-PoshGitToProfile([switch]$AllHosts, [switch]$Force, [switch]$WhatIf
         }
     }
 
+    if (!$profilePath) {
+        Write-Warning "Skipping add of posh-git import; no profile found."
+        Write-Verbose "`$profilePath          = '$profilePath'"
+        Write-Verbose "`$PROFILE              = '$PROFILE'"
+        Write-Verbose "CurrentUserCurrentHost = '${PROFILE.CurrentUserCurrentHost}'"
+        Write-Verbose "CurrentUserAllHosts    = '${PROFILE.CurrentUserAllHosts}'"

```

Comment:
Here you want `"... '$($Profile.CurrentUserAllHosts)'"`  `${}` is handy when you want to separate the variable from the next text e.g. `"${foo}${bar}"`.